scripts/src/client.ts

```diff
@@ -328,12 +338,20 @@ export class Client {
 	}
 
 	public async findKinPayment(orderId: string): Promise<CompletedPayment | undefined> {
+		if (!this.wallet) {
+			throw new Error("first set a wallet");
+		}
+
 		return (await this.wallet.getPayments())
 			.map(paymentFromTransaction)
 			.find(payment => payment !== undefined && payment.id === orderId);
 	}
 
 	public async trustKin() {
+		if (!this.wallet) {
```

Comment:
Do we even need the trust kin step still?

Reply:
it's just to pacify the compiler, if we're removing this or not can be discussed but outside of the scope of this PR